README.md

```diff
@@ -11,13 +11,7 @@ Prop supports two limiting strategies:
 To get going with Prop, you first define the read and write operations. These define how you write a registered request and how to read the number of requests for a given action. For example, do something like the below in a Rails initializer:
 
 ```ruby
-Prop.read do |key|
-  Rails.cache.read(key)
-end
-
-Prop.write do |key, value|
-  Rails.cache.write(key, value)
-end
+Prop.cache = Rails.cache # needs read/write/increment methods
```

Comment:
oh, that's in the other PR... why not base on that PR?  why not update that text here?  :confused: